modules/ROOT/pages/getting-started.adoc

```diff
@@ -1,28 +1,149 @@
 = Getting started with {ProductName}
 
-{ProductName} is a platform that helps teams develop applications. How you use {ProductName} depends on what role you have on your team. Konflux users generally fall into one of two categories:
+Adopting new platforms is always challenging. While we aim to make use of industry standard terminology and
+processes, sometimes we will need to introduce additional concepts. More definitions can be found in the
+xref:glossary:index.adoc[glossary].
 
-* Platform Engineers (PEs): These are the people whose job isn't to engineer applications, but to make it possible for others to do so. They usually own the instance of {ProductName}. They manage that instance and its users.
-* Developers:  These are the people who engineer applications for their organization. They are the typical users of {ProductName}, who productize their software with it.
+== Key concepts
 
-Depending on your role, read whichever following section applies best to you, to get more information and links to further documentation.
+=== Namespace
+A namespace in a Kubernetes cluster can have sole or shared ownership. As a user, it is the most global
+resource that is generally available to you. Most of the Kubernetes Custom Resources (CRs) which you interact
+with are scoped to a single namespace including Components, Applications, Snapshots, and Secrets.
 
-== For PEs
+All builds are performed by Tekton PipelineRuns within a namespace and usually driven by PipelineDefinitions
+committed to git repositories and triggered via https://pipelinesascode.com/[Pipelines as Code].
 
-Before anyone can use it, you need to link:https://github.com/konflux-ci/konflux-ci?tab=readme-ov-file#konflux-ci[install {ProductName}].
-//Add xref for add users and assigning them
-After installing {ProductName}, you can add users and assign them to a {ProductName} workspace. We use {ProductName} workspaces for role-based access control (RBAC).
+NOTE: {ProductName} may assume that namespaces only have one instance of some specific resources (like a secret
```

Comment:
I suggest dropping this line.

It's near the top of "getting started" and is like the third thing you learn about Konflux. It's too esoteric of a fact to be this high up in the document - or, in this document at all.
```suggestion
```